Justfile

```diff
@@ -17,15 +17,16 @@ build-python-extention:
 setup-python: build-python-extention
     cd connectorx-python && poetry run python ../scripts/python-helper.py copy-extension
     
-test-python: setup-python
-    cd connectorx-python && poetry run pytest connectorx/tests -v -s
+test-python opts="": setup-python
```

Comment:
`+opts=""` so you can input an arbitrary number of parameters after this command.